ols/app/endpoints/ols.py

```diff
@@ -108,7 +106,7 @@ def conversation_request(llm_request: LLMRequest) -> LLMRequest:
             logger.info(
                 f"{conversation_id} - Query is not relevant to kubernetes or ocp, returning"
             )
-            llm_response.response = str(
+            response = str(
```

Comment:
i guess you don't have to clean it up here, but as we refine the "response object" api, we definitely should not have a string field in our response object, into which we are putting json structured data.  If we want to have `response.detail.response="i can only answer question about......"` then we should define that overall structure of the response object.  (Note:  I do NOT think that is the overall structure we want for the response object)